File: app/src/main/java/com/jejefcgb/homelights/GPIO.java
package com.jejefcgb.homelights;

class GPIO {
    private int red;
    private int green;
    private int blue;

    public GPIO() {
    }

    public int getRed() {
        return red;
    }

    public void setRed(int red) {
        this.red = red;
    }

    public int getGreen() {
        return green;
    }

    public void setGreen(int green) {
        this.green = green;
    }

    public int getBlue() {
        return blue;
    }

    public void setBlue(int blue) {
        this.blue = blue;
    }

    @Override
    public String toString() {
        return "GPIO{" +
                "red=" + red +
                ", green=" + green +
                ", blue=" + blue +
                '}';
    }
}

File: app/src/main/java/com/jejefcgb/homelights/GPIO.kt
package com.jejefcgb.homelights

internal class GPIO {

    var red: Int = 0
    var green: Int = 0
    var blue: Int = 0

    override fun toString(): String {
        return "GPIO{" +
                "red=" + red +
                ", green=" + green +
                ", blue=" + blue +
                '}'.toString()
    }
}
